easybuild/framework/easyconfig/tweak.py

```diff
@@ -79,6 +83,9 @@ def ec_filename_for(path):
 
 def tweak(easyconfigs, build_specs, modtool, targetdirs=None):
     """Tweak list of easyconfigs according to provided build specifications."""
+    # keep track of originally listed easyconfigs (via their filename)
+    listed_ec_filenames = [os.path.basename(ec['spec']) for ec in easyconfigs]
```

Comment:
@boegel Is this really wise actually? You just mentioned a use case where the file name does not correspond to what is in the file, if we don't keep the entire path then we have lost uniqueness

Reply:
Right, I overlooked that...

But with the changes you made, full path is also not a good idea, since you use the location of the files and make assumptions.

Reply:
I don't think I do any more (I use `alt_easyconfig_paths()` now) so I believe I can switch this back